### DISCUSSION TITLE
Dialogs customization

### DISCUSSION THREAD

**Discussion by ghost** [taiga-family/taiga-ui#967] [Q&A] [answered]
Подскажите пожалуйста, как я могу изменить классы диалога?

![image](https://user-images.githubusercontent.com/44188728/140011756-4325a6a4-01a1-4cf8-a9ac-c387e30246b2.png)

Мне необходимо удалить `border` и изменить `border-radius`.
И в классе `content`:  `border-radius: inherit; padding: 2rem; background: var(--tui-base-01);`, необходимо изменить `padding`.

**Comment by nsbarsukov** [comment]
Hi!
it is easy to create your own type of dialog: with your paddings, colors, etc.

Look at this page: https://taiga-ui.dev/components/dialog/Your_own_dialogs 

1. create [identical service](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/core/components/dialog/dialog.service.ts) with **your** dialog component
2. provide it into multi token `TUI_DIALOGS`

**Comment by ghost** [comment, marked as answer]
@nsbarsukov у вас имеется пример пользовательского окна на stackblitz представленный на странице [https://taiga-ui.dev/dialogs](https://taiga-ui.dev/dialogs), но он к сожалению не работает. Поправьте пожалуйста, чтобы можно было разобраться с данной реализацией

**Reply by nsbarsukov** [reply]
it will be fixed soon https://github.com/TinkoffCreditSystems/taiga-ui/pull/969

**Reply by ghost** [reply]
@nsbarsukov К сожалению пример [пользовательского диалога](https://taiga-ui.dev/dialogs) на stackblitz по прежнему не работает
